tests/cxx/unit_tests/ghostfragment/capping/dclc.cpp

```diff
@@ -0,0 +1,61 @@
+#include "../test_ghostfragment.hpp"
+#include "DCLC.hpp"
+#include <ghostfragment/property_types/connectivity_table.hpp>
+#include <hydrocarbon/hydrocarbon_fragment.hpp>
+
+TEST_CASE("DCLC Capping") {
+
+    using namespace ghostfragment;
+    using namespace testing;
+
+    using the_pt       = pt::Capped;
+    using input_type   = chemist::FragmentedNuclei;
+    using result_type  = std::vector<chemist::CapSet>;
+    using conn_pt      = ghostfragment::ConnectivityTable;
+
+    auto mm        = initialize();
+    auto& mod      = mm.at("DCLC Capping");
+
+    SECTION("Ethane (2 carbon 2 frags)") {
```

Comment:
Sorry I forgot to submit this comment on the last review, but you should also add a test case for methane (1 carbon) to make sure that your module works when the system doesn't actually need capping.

Reply:
Added test case for methane.